src/app/src/main/kotlin/org/vaccineimpact/api/app/logic/CoverageLogic.kt

```diff
@@ -48,7 +48,8 @@ class RepositoriesCoverageLogic(private val modellingGroupRepository: ModellingG
             repositories.scenario,
             repositories.expectations)
 
-    private val expectedGAVICoverageYears = LocalDate.now().plusYears(1).year..2030
+    private val mandatoryGAVICoverageYears = 2021..2030
```

Comment:
Obviously this will make 2021 mandatory even when we're in 2022... presumably that's ok?

Reply:
I think that's ok for now, it does mean we'll need to update it next year. No doubt other things will have changed by then!

Reply:
No problem - the alternative (i.e. current year to 2030) may not actually be better.